tcms/core/tests/test_markdown.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding: utf-8 -*-
+import unittest
+import bleach
+
+from tcms.core.templatetags.extra_filters import markdown2html
+
+
+class TestMarkdownExtraFilters(unittest.TestCase):
+
+    def setUp(self):
+        self.md_simple_text = markdown2html("__*hello!*__")
+        self.md_table = markdown2html("""|Stx |Desc |\n|----|-----|\n|Head|Title|\n|Text|Text |""")
+        self.md_nl2br = markdown2html("""Line 1
+Line 2""")
+        self.md_fenced_code = markdown2html("""```{

```

Comment:
the naming of these variables is wrong. `md` implies markdown, but this is not markdown, this is markdown, converted to html